src/test/java/com/jwplayer/jwplatform/client/AdvertisingClientTest.java

```diff
@@ -0,0 +1,50 @@
+package com.jwplayer.jwplatform.client;
+
+import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.json.JSONObject;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import com.jwplayer.jwplatform.exception.JWPlatformException;
+import com.jwplayer.jwplatform.rest.HttpCalls;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({HttpCalls.class})
+public class AdvertisingClientTest {
```

Comment:
You'll want to add a test for a non-200 status which then throws an exception.

Example from the v1 client: https://github.com/jwplayer/jwplatform-java/blob/master/src/test/java/com/jwplayer/jwplatform/TestJWPlatformClient.java#L115

Comment applies elsewhere.